### ISSUE TITLE
Running CorsixTH on Windows 10 Problem

### ISSUE THREAD

**Issue by ZaiHuo** [CorsixTH/CorsixTH#1404]
#### Describe the issue

Hello,

I'm a CorsixTH user for a long time and have been successfully running it on Windows (up to 7) beautifully without any issues.

Upon switching to Windows 10 very recently I was unable to run it any longer. CorsixTH would open normally, play the intro video, then crash a few moments after the Main Menu of CorsixTH appears.

Windows would display a "CorsixTH has stopped working" window, and there were absolutely no hints of what was causing the issue on the report window (both Powershell and Extra Window) and the error was not generating a gamelog.txt. The problem persisted on the following CorsixTH windows installations and Theme Hospital versions:

- Standard Windows CorsixTH v.061 installation
- Portable Windows CorsixTH v.061 installation
- Origin's Original Theme Hospital Installation
- GOG's Theme Hospital
- Microsoft Visual C++ Redistributable (x64) - 14.14.26405 installed
- Both local and appdata configuration locations chosen during installation

#### Steps to Reproduce

After making all the local tests I could and still being unable to run CorsixTH I looked for help at the IRC channel, where sadgerino suggested me disable as much as I could to possibly find the issue. 

After disabling the audio my CorsixTH worked normally again and the game is playable normally, without audio.

#### System Information
*CorsixTH Version: v.061

**Operating System: Windows 10 64 bits

**Theme Hospital Version:** CD, GOG.com, Origin.

#### Gamelog.txt

None

### Additional Info

No audio hardware/driver outdated or running/incompatibility problems, and the device isn't even a week old.

**Comment by wolfy1339** [comment]
Strange. I run it on Windows 10 and haven't experienced that problem

**Comment by MarkL1961** [comment]
Windows 10 Home here and no problems either

**Comment by MarkL1961** [comment]
What audio  system do you have?
Are you using original sound files or other?

**Comment by ZaiHuo** [comment]
The devs suspected it was the drivers, since the device was 'too new' it could have a weird version of the driver installed, and that was exactly the case, I went to the manufacturer's website (Vaio) and downloaded/installed the audio version that was there.

Funnily enough I didn't 'upgrade' the driver, but 'downgraded' it, the version at the website was actually older by more than a year, but.... surprisingly, it worked, it's running normally now after downgrading the audio driver.

Thank you very much for the devs's advice at the irc channel and the quick replies from the community, thanks to your directions I managed to make it work normally. Now I can go back to no-lifing CorsixTH.
